### ISSUE TITLE
OBI:0100026 as taxon for variant objects

### ISSUE THREAD

**Issue by deepakunni3** [monarch-initiative/monarch-ui#242]
This bug was originally raised by @iimpulse 

For reference, the query: https://api-dev.monarchinitiative.org/api/bioentity/gene/MGI:98297/variants?fetch_objects=true&start=0&rows=10&facet=true&facet_fields=subject_taxon&taxon=OBI%3A0100026

yields gene to variant associations. But there are some associations where object has a `BNODE` prefix, and a taxon of `OBI:0100026 'organism'`. This makes it difficult to filter variants based on taxon since 'organism' is too generic of a taxon term.

@cmungall @kshefchek Thoughts?

CC'ing @monicacecilia for her awesomeness!

**Comment by kshefchek** [comment]
we originally attempted to infer taxon on genotype parts, instead of making an explicit edge for each (in retrospect maybe a mistake).  Theres an inference path in the solr loader that has been broken for some time, in the sense that it either doesn't infer the taxa or infers 'organism'.

tl;dr this should probably be fixed in dipper and likely won't be in the short term

**Comment by kshefchek** [comment]
see also - https://github.com/SciGraph/golr-loader/issues/10

**Comment by deepakunni3** [comment]
Thats good to know.

Thanks @kshefchek 

So this is blocked by Dipper or SciGraph loader? or both?

**Comment by kshefchek** [comment]
The way it works now, this could either be fixed in dipper or the golr-loader code.  We could also add something in scigraph but it would be some new post processor.  I think the best thing to do is to add it in dipper.
